### DISCUSSION TITLE
Why my W7900 GPU build RAM is being used ?

### DISCUSSION THREAD

**Discussion by EngrAwab** [ROCm/ROCm#3879] [Q&A]
I wanted to fine tune the LLM model I tried at colab it ran good without any issue and then I tried to reproduce that at my own system but I was unable to reproduce it because my ram was 32GB and upon loading my system get freeze. The main thing I am concern about is that in colab the RAM was less than 32 GB and still I was able to complete that but why not here ?

**Comment by huanrwan-amd** [comment]
Hi @EngrAwab, can you provide more details on your system: software version (ROCm, OS) and hardware detail? A minium reproduciable code is also helpful. Thanks.

**Reply by EngrAwab** [reply]
I am running Ubuntu 22.04 LTS and ROCM is 6.2.... RAM I am using is 6000MHz CL30  16GB*2

**Reply by huanrwan-amd** [reply]
Hi @EngrAwab, can you please provide further details on your side?

1. rocm-smi for showing GPU usage.
2. rocminfo for gived details on the hardware
3. On fine tuning LLM side, can you please start with a smaller model?
A minimum reproducible code is also helpful. Thanks.
